handlers.py

```diff
@@ -72,6 +72,27 @@ def get_all_responses_with_game(self, message, game):
             return []
 
 
+class SameHandler(GameExtractionMixin, ContentBasedHandler):
+    fragments = ['same', 'Same']
+
+    def get_all_responses_with_game(self, message, game):
+        would_play = []
+        fragment_present = any(message.content.lower().startswith(f.lower()) for f in self.fragments)
+        last_would_play = db.get_last_would_play(self.game_name)
+        
+        if not last_would_play:
+            return ['Error! No fun here yet!']
+
+        if self.game_name and fragment_present:
+            would_play = db.record_would_play(message.author, game) 
```

Comment:
You record would plays regardless as long as fragment_present is true, so perhaps it should be extracted (might not matter based on what I said above about fragments being required for this to even run). I feel like this whole chunk might be able to benefit from something like `game = self.game if self.game else last_would_play.game` or the shorter `game = self.game or last_would_play.game`